lib/zuora/objects/subscription.rb

```diff
@@ -12,18 +12,18 @@ class Subscription < Base
                           :renewal_term
 
     validates_inclusion_of    :auto_renew, :in => [true, false]
-    validates_datetime_of     :cancelled_date, :allow_nil => true
-    validates_datetime_of     :contract_acceptance_date, :allow_nil => true
-    validates_datetime_of     :contract_effective_date
+    # validates_datetime_of     :cancelled_date, :allow_nil => true
+    # validates_datetime_of     :contract_acceptance_date, :allow_nil => true
+    # validates_datetime_of     :contract_effective_date
     validates_numericality_of :initial_term, :only_integer => true, :minimum => 1
     validates_inclusion_of    :is_invoice_separate, :in => [true, false], :allow_nil => true
     validates_length_of       :name, :maximum => 100
     validates_length_of       :notes, :maximum => 500, :allow_nil => true
-    validates_datetime_of     :original_created_date, :allow_nil => true
+    # validates_datetime_of     :original_created_date, :allow_nil => true
     validates_numericality_of :renewal_term, :only_integer => true
-    validates_datetime_of     :service_activation_date, :allow_nil => true
-    validates_datetime_of     :term_end_date, :allow_nil => true
-    validates_datetime_of     :term_start_date
+    # validates_datetime_of     :service_activation_date, :allow_nil => true
+    # validates_datetime_of     :term_end_date, :allow_nil => true
+    # validates_datetime_of     :term_start_date
```

Comment:
Do we know why these are now commented out?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)